tasks/glance_image.yml

```diff
@@ -1,5 +1,12 @@
 ---
 - block:
+  - name: Check if template is correct
+    fail:
+      msg: "mandatory parameter '{{ item }}' is not defined"
+    when: "{{ item }} is undefined"
```

Comment:
Here as well, is the `{{ item }}` right, or should it be just `item`, doesn't it produce any warning messages?